app/org/sagebionetworks/bridge/models/studies/StudyAndUserHolder.java

```diff
@@ -0,0 +1,31 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.List;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.sagebionetworks.bridge.models.BridgeEntity;
+import org.sagebionetworks.bridge.models.accounts.StudyParticipant;
+
+public final class StudyAndUserHolder implements BridgeEntity {
```

Comment:
I think you can just call this StudyWithUsers or StudyAndUsers. "Holder" as a word is reserved for key-holding objects that replace full entities, which this is not.